README.md

```diff
@@ -1,24 +1,73 @@
-# Cartesi Rollups
+# Smart Contracts for Cartesi Rollups
 
-This repository contains the on-chain and off-chain pieces that are used to deploy, launch and interact with Cartesi Rollups DApps. The code presented here is work in progress, continuously being improved and updated.
+This repository contains the on-chain part of Cartesi Rollups.
+
+If you are interested in taking a look at the off-chain part, please, head over to the [Rollups Node](https://github.com/cartesi/rollups-node) repository.
+
+## Dependencies
+
+- [Yarn](https://classic.yarnpkg.com/en/docs/install)
+- [Forge](https://book.getfoundry.sh/getting-started/installation)
+- [Docker](https://docs.docker.com/get-docker/)
+
+## Local setup
+
+This repository contains [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
+In order to properly initialize them, please, run the following command.
+
+```sh
+git submodule update --init --recursive
+```
+
+This repository uses [Yarn](https://classic.yarnpkg.com/en/docs/install) to manage npm packages.
+In order to install them, please, run the following commands.
+
+```sh
+cd onchain
+yarn
+```
+
+_(Optional)_ If you plan to run the on-chain tests, there still some setup left to do.
+Assuming you are on the `onchain` directory, you may run the following commands.
+
+```sh
+cd rollups
+yarn proofs:setup
```

Comment:
should we refer to https://github.com/cartesi/rollups-contracts/tree/feature/adapt-readme/onchain/rollups/test/foundry/dapp/helper here?

Reply:
I think it's better to move the documentation from there to here, actually.
This is a crucial part for running tests, so it's good to have it all in once place.

Reply:
Agree